### PR TITLE
docs(core): time examples

### PR DESCRIPTION
**Pull request by web-dev23**
#### Please provide a link to the associated issue.
Closes https://github.com/SAP/fundamental-ngx/issues/4391

#### Please provide a brief summary of this pull request.
Fixes Time component Stackblitz examples.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by Betrozov — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/time/examples/time-form-example.component.ts [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import { Component } from '@angular/core';
 import { FormControl, FormGroup } from '@angular/forms';
+import { DatetimeAdapter, FdDatetimeAdapter } from '@fundamental-ngx/core';
```
Comment (N1XUS) [inline apps/docs/src/app/core/component-docs/time/examples/time-form-example.component.ts:3]: Please import from '@fundamental-ngx/core/datetime'

File: apps/docs/src/app/core/component-docs/time/examples/time-12-example.component.ts [hunk 2]
```diff
@@ -1,9 +1,16 @@
 import { Component } from '@angular/core';
-import { FdDate } from '@fundamental-ngx/core/datetime';
+import { DatetimeAdapter, DATE_TIME_FORMATS, FdDate, FdDatetimeAdapter } from '@fundamental-ngx/core/datetime';
+import { CUSTOM_FD_DATETIME_FORMATS } from 'apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-format-example.component';
```
Comment (dimamarksman) [inline apps/docs/src/app/core/component-docs/time/examples/time-12-example.component.ts:3]: unused variable, please drop it

File: apps/docs/src/app/core/component-docs/time/examples/time-example.component.ts [hunk 3]
```diff
@@ -1,9 +1,16 @@
 import { Component } from '@angular/core';
-import { FdDate } from '@fundamental-ngx/core/datetime';
+import { DatetimeAdapter, DATE_TIME_FORMATS, FdDate, FdDatetimeAdapter } from '@fundamental-ngx/core/datetime';
+import { CUSTOM_FD_DATETIME_FORMATS } from 'apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-format-example.component';
```
Comment (dimamarksman) [inline apps/docs/src/app/core/component-docs/time/examples/time-example.component.ts:3]: unused variable, please drop it

File: apps/docs/src/app/core/component-docs/time/examples/time-12-example.component.ts (hunk 2 above)
Comment (web-dev23) [reply apps/docs/src/app/core/component-docs/time/examples/time-12-example.component.ts:3]: Fixed.

File: apps/docs/src/app/core/component-docs/time/examples/time-example.component.ts (hunk 3 above)
Comment (web-dev23) [reply apps/docs/src/app/core/component-docs/time/examples/time-example.component.ts:3]: Fixed.

File: apps/docs/src/app/core/component-docs/time/examples/time-form-example.component.ts (hunk 1 above)
Comment (web-dev23) [reply apps/docs/src/app/core/component-docs/time/examples/time-form-example.component.ts:3]: Fixed.

**Pull request merged**